File: LeetcodeStuff/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)-1):
            t = (nums[i]+nums[i+1])
            if (t) == target:
                ans = []
                ans.append(i)
                ans.append(i+1)
                return ans

File: LeetcodeStuff/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector <int> ans;
        for(int i =0;i<nums.size() -1;i++){
            int t = nums[i] + nums[i+1];
            if (t == target){
                ans.push_back(i);
                ans.push_back(i+1);
            }
        }
        return ans;
    }
};
